### ISSUE TITLE
Encoding problem during `pip install`

### ISSUE THREAD

**Issue by Noordsestern** [hiveeyes/dwdweather2#5]
System:
- Windows 10
- Python 3.7

I run in to the following problem when running `pip install dwdweather2`:
```
PS D:\workspaces\robotframework-dwdweather> pip install dwdweather2
Collecting dwdweather2
  Using cached https://files.pythonhosted.org/packages/b9/0c/fb48f9e43f263a493f0a5e13d00bbe6ca0a31d11958b444836d218951a25/dwdweather2-0.8.0.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Daidalos\AppData\Local\Temp\pip-install-pa4hwo8d\dwdweather2\setup.py", line 6, in <module>
        README = open(os.path.join(here, 'README.rst')).read()
      File "d:\programs\python\python37\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 2062: character maps to <undefined>
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in C:\Users\Daidalos\AppData\Local\Temp\pip-install-pa4hwo8d\dwdweather2\
```
Installation works with `dwdweather` (is this project related to you) but not with `dwdweather2`.

Installing `dwdweather`:
```
PS D:\workspaces\robotframework-dwdweather> pip install dwdweather
Collecting dwdweather
  Downloading https://files.pythonhosted.org/packages/ee/4f/4bd3ceff7b6158133d6e7cbfd68f27e4d72fd2f80c83d032944e16bbbb2f/dwdweather-0.7.tar.gz
Installing collected packages: dwdweather
  Running setup.py install for dwdweather ... done
Successfully installed dwdweather-0.7
```

**Comment by amotl** [comment]
Dear @Noordsestern,

thanks for writing in.

> `UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 2062: character maps to <undefined>`

Ah, the usual suspects if ``README.rst`` is not totally ASCII-clean.

Thanks already for providing a fix through #6, we just released version 0.8.2 which should mitigate this issue for both Python 2.7 and Python 3.7.

With kind regards,
Andreas.

---

P.S.:

> Is [the [dwdweather](https://github.com/marians/dwd-weather)] project related to you?

We just took over the further development from there, so `dwdweather2` is actually a descendant of `dwdweather`. See also https://github.com/marians/dwd-weather/issues/3.
